astropy/modeling/blackbody.py

```diff
@@ -48,6 +48,26 @@
 >>> flux_nu  # doctest: +FLOAT_CMP
 <Quantity [  4.25135927e-123,  2.36894060e-005] erg / (cm2 Hz s sr)>
 
+Alternatively, the same results for ``flux_nu`` can be computed using ``BlackBody1D``
+from :class:`BlackBody1D` with blackbody representation as a model. The difference between
+this and the former approach is in one addition step as outlined as follows:
+
+>>> import numpy as np
+>>> from astropy import constants as const
+>>> from astropy import units as u
+>>> from astropy.modeling import models
+>>> temperature = 5000 * u.K
+>>> extra = const.sigma_sb * temperature ** 4 / np.pi
```

Comment:
The variable name `extra` is too cryptic; extra what? If you cannot think of something else, I guess we can fall back to using `bolometric_flux`, then the call to `BlackBody1D` would have `bolometric_flux=bolometric_flux`.

Reply:
Sorry, I meant the "extra" step involved in the calculation... Will change that to `bolometric_flux` soon.